Gemfile.lock

```diff
@@ -3,7 +3,7 @@ GEM
   specs:
     ansi (1.5.0)
     ast (2.4.2)
-    beaker (4.36.1)
+    beaker (4.38.1)
```

Comment:
Ultimately I needed to update `net-ssh` because of openssl version 3 not liking some immutable keys (after the systemd update).